src/bin/init/src/main.rs

```diff
@@ -261,13 +261,34 @@ fn main() {
         ObjectCreateFlags::empty(),
     );
     let netid = twizzler_abi::syscall::sys_object_create(create, &[], &[]).unwrap();
-    /*
+    let devid = twizzler_abi::syscall::sys_object_create(create, &[], &[]).unwrap();
+    println!("starting device manager");
```

Comment:
General question abou print calls throughout the system.  Do you want to switch to the logging crate?  Given we're an OS somehow that seems more appropriate going forwards, especially since it gives finer grained control over when we get console messages.

Reply:
Yes, logging is one of the major things I want to work on once the initial driver stuff is done. For now, I'm leaving things in as println, but once we have logging, I intend to audit all of these calls.